xtask/codegen/css.ungram

```diff
@@ -399,15 +399,17 @@ CssBlock =
 	declaration_list: CssDeclarationList
  	'}'
 
-
-CssDeclarationList = CssDeclaration*
+CssDeclarationList = CssDeclaration (';' CssDeclaration)* ';'?
 
 CssDeclaration =
-	name: CssIdentifier | CssCustomProperty
+	name: CssDeclarationName
 	':'
-	value: AnyCssValue
+	value: CssListOfComponentValues
 	important: CssDeclarationImportant?
 
+CssListOfComponentValues = AnyCssValue*
+
+CssDeclarationName = CssIdentifier | CssCustomProperty
```

Comment:
It's better to rename to `AnyCssDeclarationName`

Reply:
done